_plugins/dashboard.rb

```diff
@@ -43,6 +43,13 @@ def self.munge_project_data(project_data)
       FIELDS_TO_TRANSLATE.each do |from, to|
         project_data[to] = project_data[from] unless project_data[to]
       end
+      original_name = project_data['name']
+      if original_name.scan(/[A-Z]/).size > 0
+        project_data['redirect_from'] = Array.new
+        project_data['redirect_from'].push("project/#{original_name}")
```

Comment:
_I_ wouldn't thought to do that because I'm not as good at reading or writing idomatic ruby and don't fully understand that line at a quick glance. But from what I _do_ understand of ruby's idioms that looks good.